### DISCUSSION TITLE
Multiple "negotiated_rate"s in Single "negotiated_prices" Object - How to Interpret?

### DISCUSSION THREAD

**Discussion by medanalytics** [CMSgov/price-transparency-guide#735] [General]
@shaselton-usds  In the latest CareFirst in-network files, there are multiple rates in a single negotiated_prices object, with no other fields being different, as shown below:
![image](https://github.com/CMSgov/price-transparency-guide/assets/160523556/a0172361-5060-4b40-bdff-827ec7caacb4)
Note: although the service_code arrays may look different, they have the same codes ordered differently.
This is the corresponding provider_groups object:
![image](https://github.com/CMSgov/price-transparency-guide/assets/160523556/746659fc-1e96-431c-92bb-37c85e6824d8)

How should this information be interpreted? How can I know which price is the actual price?
I suggest ensuring that the schema allows for only one rate when all other fields are effectively the same.

**Comment by shaselton-usds** [comment]
I would assume that there would be a difference in the composite data that makes up the negotiated price. Have you tried reaching out to the vendor producing the MRFs to get their thoughts?

**Reply by medanalytics** [reply]
@shaselton-usds when you say composite data, what do you mean? This seems to have been unresolved on this discussion thread as well (#633)

**Reply by shaselton-usds** [reply]
Hi @medanalytics Sorry for the confusion. What I mean with "composite data" is the unique combination of the data elements.
